perl/perl.bzl

```
@@ -102,7 +102,10 @@ def transitive_deps(ctx, extra_files = [], extra_deps = []):
 
 def _include_paths(ctx):
     """Calculate the PERL5LIB paths for a perl_library rule's includes."""
-    package_root = (ctx.label.workspace_root + "/" + ctx.label.package).strip("/") or "."
+    workspace_root = ctx.label.workspace_root
+    if workspace_root.startswith("external/"):
+        workspace_root = "../" + workspace_root[9:]
+    package_root = (workspace_root + "/" + ctx.label.package).strip("/") or "."
```

```suggestion
    workspace_name = ctx.label.workspace_name
    if workspace_name:
        workspace_root = "../" + workspace_name
    else:
        workspace_root = ""
    package_root = (workspace_root + "/" + ctx.label.package).strip("/") or "."
```

may be a bit cleaner as it doesn't have to get rid of `external`.